ModelBundle/Migrations/MongoDB/Version20160720134051.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Doctrine\MongoDB\Database;
+
+/**
+ * Class Version20160720134051
+ */
+class Version20160720134051 extends AbstractMigration
+{
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return "Update node with new structure";
+    }
+
+    /**
+     * @param Database $db
+     */
+    public function up(Database $db)
+    {
+        $db->execute('
+            var updateArea = function(area) {
```

Comment:
Same method and same exploitation as in node. Maybe it could be factorize in a areaContainer Migration.